aws/resource_aws_ecs_task_definition_test.go

```diff
@@ -649,7 +654,7 @@ TASK_DEFINITION
 `, tdName)
 }
 
-func testAccAWSEcsTaskDefinitionFargate(tdName string) string {
+func testAccAWSEcsTaskDefinitionFargate(tdName, portMappings string) string {
```

Comment:
Nitpick: It'd be much better if this was a separate test and test configuration as it does not pertain to the basic functionality of the resource with Fargate